### PR TITLE
add org.wownero.wownero-wallet-cli

### PR DESCRIPTION
**Pull request by wowario**
#860

### CODE REVIEW

**Comment by wowario** [comment]
bot, build org.wownero.wownero-wallet-cli
testbot, build org.wownero.wownero-wallet-cli

**Comment by wowario** [comment]
bot, build org.wownero.wownero-wallet-cli
testbot, build org.wownero.wownero-wallet-cli

**Comment by wowario** [comment]
bot, build org.wownero.wownero-wallet-cli
testbot, build org.wownero.wownero-wallet-cli

File: org.wownero.wownero-wallet-cli.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 wowario -->
+<component type="desktop-application">
```
Comment (barthalion) [inline org.wownero.wownero-wallet-cli.appdata.xml]: It should be `console-application`. Also appdata misses OARS ratings.

File: org.wownero.wownero-wallet-cli.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "app-id": "org.wownero.wownero-wallet-cli",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "/app/bin/wownero-wallet-cli",
```
Comment (barthalion) [inline org.wownero.wownero-wallet-cli.json]: `/app/bin` is in the default path so `wownero-wallet-cli` should be enough.

File: org.wownero.wownero-wallet-cli.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      "build-commands": [
+        "ar x wownero_0.5.0.0_amd64.deb",
+        "tar -xf data.tar.xz",
+        "install -Dm755 usr/local/bin/wownero-wallet-cli /app/bin/wownero-wallet-cli",
+        "install -Dm644 org.wownero.wownero-wallet-cli.appdata.xml /app/share/appdata/org.wownero.wownero-wallet-cli.appdata.xml"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "url": "http://ppa.wownero.com/pool/main/w/wownero/wownero_0.5.0.0_amd64.deb",
```
Comment (barthalion) [inline org.wownero.wownero-wallet-cli.json]: It should be build from source.

**Comment by wowario** [comment]
building from source and combining with wownerod pr

**Pull request closed without merging**